### PR TITLE
Synchronize accept param with accept of content

### PR DESCRIPTION
**Pull request by fkroockmann**
Now we can change directly the accept in parameter of content.
When a param "accept" is saved the accept of content is synchronized with the param

### CODE REVIEW

File: ClassContent/ContentSet.php [hunk 1]
```diff
@@ -601,6 +601,94 @@ protected function defineData($var, $type = 'scalar', $options = null, $updateAc
...
+     *
+     * @param string  $types
+     * @return array full classname array
+     */
+    protected function addClassname($types = array())
+    {
+        $types = (array) $types;
+
+        array_walk($types, function(&$item) {
+            if ('!' === substr($item, 0, 1)) {
```
Comment (eric-chau) [inline ClassContent/ContentSet.php]: you can avoid a function call by using `$item[0]` =)

Comment (fkroockmann) [inline ClassContent/ContentSet.php]: COol :D

File: ClassContent/ContentSet.php [hunk 2]
```diff
@@ -601,6 +601,94 @@ protected function defineData($var, $type = 'scalar', $options = null, $updateAc
     /**
      * {@inheritdoc}
      */
+    public function setParam($key, $value = null)
+    {
+        parent::setParam($key, $value);
+
+        if ('accept' === $key && is_array($value)) {
+
```
Comment (eric-chau) [inline ClassContent/ContentSet.php]: This line can be removed

File: ClassContent/ContentSet.php [hunk 3]
```diff
@@ -601,6 +601,94 @@ protected function defineData($var, $type = 'scalar', $options = null, $updateAc
...
+        return $param;
+    }
+
+    /**
+     * Returns the short classname of several types.
+     *
+     * @param string  $types
+     * @return array short classname array
+     */
+    protected function removeClassname($types = array())
```
Comment (eric-chau) [inline ClassContent/ContentSet.php]: `convertToContentType()` would be more appropriate no?

File: ClassContent/ContentSet.php [hunk 4]
```diff
@@ -601,6 +601,94 @@ protected function defineData($var, $type = 'scalar', $options = null, $updateAc
...
+        return $types;
+    }
+
+    /**
+     * Returns the full classname of several types.
+     *
+     * @param string  $types
+     * @return array full classname array
+     */
+    protected function addClassname($types = array())
```
Comment (eric-chau) [inline ClassContent/ContentSet.php]: `convertToClassname()` would be more appropriate no?

**Comment by eric-chau** [comment]
:+1: good job thank you @fkroockmann

**Comment by fkroockmann** [comment]
All done, thanks for review @eric-chau

File: ClassContent/ContentSet.php [hunk 5]
```diff
@@ -617,7 +703,7 @@ protected function defineParam($var, $options = null)
 
     /**
      * Adds a new accepted type to the element.
-     *
+     *convertToContentType
```
Comment (eric-chau) [inline ClassContent/ContentSet.php]: This is a wrong copy/paste?

**Comment by crouillon** [comment]
:+1: Thanks @fkroockmann

**Pull request merged**